easybuild/tools/toolchain/compiler.py

```diff
@@ -93,6 +93,7 @@ class Compiler(Toolchain):
         'vectorize': (None, "Enable compiler auto-vectorization, default except for noopt and lowopt"),
         'packed-linker-options': (False, "Pack the linker options as comma separated list"),  # ScaLAPACK mainly
         'rpath': (True, "Use RPATH wrappers when --rpath is enabled in EasyBuild configuration"),
+        'search-path-cpp-headers': (False, "Search path used at build time for include directories"),
```

Comment:
good point, this needed a bit more changes to avoid warnings in the logs, fully fixed in https://github.com/easybuilders/easybuild-framework/pull/4645/commits/324c79c1cbef9ba200e974d5af2ff12b579f47b2